Sources/SotoCore/Doc/Region.swift

```diff
@@ -167,3 +167,70 @@ extension AWSPartition {
         }
     }
 }
+
+// allows to create a Region from a String
+// this initialializer returns an Optional.
+// it is non nil for valid region name only
+public extension Region {
+    init?(regionName: String) {
+        switch regionName {
+        case "af-south-1":
+            self = Region.afsouth1
```

Comment:
Can we change this to
```swift
        self.init(rawValue: regionName)
        switch self {
        case .euwest1,
             .euwest2,
                 ...
             return
        default:
            return nil

...
```

Reply:
The problem with this is Stencil. The last line finishes with `.euwest2 :` (colon and not coma)
Is there a way to tell Stencil to loop on all occurences, except the last one ?

Reply:
You can use `{%if not forloop.last %}`

Reply:
done